lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,27 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let kTypeStrippingMode = null;
+ObjectDefineProperty(process.features, 'typescript', {
+  __proto__: null,
+  get() {
+    if (kTypeStrippingMode === null) {
```

Comment:
Maybe you can do something like this:
```
const hasStripTypes = getOptionValue('--experimental-strip-types');
const hasTransformTypes = getOptionValue('--experimental-transform-types');

ObjectDefineProperty(process.features, 'typescript', {
  __proto__: null,
  get() {
    if (!hasStripTypes) {
      return;
    }
    return hasTransformTypes ? 'transform' : 'strip';
  },
  configurable: false,
  enumerable: true,
});
```

Reply:
Wouldn't that be slower, as the value isn't stored anywhere?

Reply:
Maybe, but I dont think its a relevant difference, feel free to ignore